invenio_rdm_records/contrib/meeting/custom_fields.py

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Meeting custom fields.
+
+Implements the following fields:
+
+- meeting.acronym
+- meeting.dates
+- meeting.place
+- meeting.session_part
+- meeting.session
+- meeting.title
+- meeting.url
+"""
+
+from flask_babelex import _
+from invenio_records_resources.services.custom_fields import BaseCF
+from marshmallow import fields, validate
+from marshmallow_utils.fields import SanitizedUnicode
+
+
+class MeetingCF(BaseCF):
+    """Nested custom field."""
+
+    @property
+    def field(self):
+        """Meeting fields definitions."""
+        return fields.Nested(
+            {
+                "acronym": SanitizedUnicode(),
+                "dates": SanitizedUnicode(),
+                "place": SanitizedUnicode(),
+                "session_part": SanitizedUnicode(),
+                "session": SanitizedUnicode(),
+                "title": SanitizedUnicode(),
+                "url": SanitizedUnicode(
+                    validate=validate.URL(error="You must provide a valid URL."),
+                ),
+            }
+        )
+
+    @property
+    def mapping(self):
+        """Meeting search mappings."""
+        return {
+            "type": "nested",
+            "properties": {
+                "acronym": {"type": "keyword"},
+                "dates": {"type": "keyword"},
+                "place": {"type": "text"},
+                "session_part": {"type": "keyword"},
+                "session": {"type": "keyword"},
+                "title": {"type": "text"},
+                "url": {"type": "keyword"},
+            },
+        }
+
+
+MEETING_NAMESPACE = {
+    # Meeting
+    "meeting": "",
+}
+
+
+MEETING_CUSTOM_FIELDS = [
+    MeetingCF(name="meeting:meeting"),
+]
+
+MEETING_CUSTOM_FIELDS_UI = {
+    "section": _("Conference"),
+    "fields": [
+        {
+            "field": "meeting:meeting",
+            "ui_widget": "Meeting",
+            "template": "meeting.html",
+            "props": {
+                "label": _("Conference"),
+                "title": {
+                    "label": _("Conference title"),
+                    "placeholder": _(""),
+                    "description": _("Add the conference title"),
```

Comment:
This was also commented in another PR. We need a consensus on how we do descriptions:
a) do we put "actionables" (i.e. add, write, fill in, etc.)
b) do we put mere descriptive text (e.g. "Title of the conference")

I think in the current deposit form we have b). I'm not against changing it, just to be consistent.

Reply:
I agree on keeping b) , for consistency.